docs/versioned_docs/version-2.11.0/deprecated/ugcstor.md

```diff
@@ -0,0 +1,2496 @@
+```
+
+Note: `SNAPSHOT_LOCATION`  should be the same as you configured by using `06-volumesnapshotlocation.yaml`
+
+Get the details of backup using the following command
+
+```
+velero backup get
+```
+
+During the first backup iteration of a schedule, full data of the volume will be backed up. After taking the full backup in the first schedule, then it will take the incremental backup as part of the next iteration.
+
+#### Restore from a Scheduled Backup
+
+Since the backups taken are incremental for a schedule, order of restoring data is important. You need to restore data in the order of the backups created.
+
+For example, below are the available backups for a schedule.
+
+```
+NAME                   STATUS      CREATED                         EXPIRES   STORAGE LOCATION   SELECTOR
+sched-20190513104034   Completed   2019-05-13 16:10:34 +0530 IST   29d       gcp                <none>
+sched-20190513103534   Completed   2019-05-13 16:05:34 +0530 IST   29d       gcp                <none>
+sched-20190513103034   Completed   2019-05-13 16:00:34 +0530 IST   29d       gcp                <none>
+```
+
+Restoration of data need to be done in following way:
+
+```
+velero restore create --from-backup sched-20190513103034 --restore-volumes=true
+velero restore create --from-backup sched-20190513103534 --restore-volumes=true
+velero restore create --from-backup sched-20190513104034 --restore-volumes=true
+```
+
+#### Deletion of Backups
+
+To delete a single backup which is not created from scheduled backup, use the following command.
+
+```
+velero backup delete <backup_name>
+```
+
+Note: the deletion of backup will not delete the snapshots created as part of backup from the cStor Storage pool. This can be deleted by following manual steps .
+
+1. First verify the cStor backups created for corresponding cStor volume. To obtain the cStor backups of cStor volume, use the following command by providing the corresponding backup name.
+
+   ```
+   kubectl get  cstorbackups  -n <backup_namespace> -l openebs.io/backup=<backup_name>  -o=jsonpath='{range .items[*]}{.metadata.labels.openebs\.io/persistent-volume}{"\n"}{end}'
+   ```
+
+2. Delete the corresponding cStor backups using the following command.
+
+   ```
+   kubectl delete cstorbackups -n <backup_namespace> -l openebs.io/backup=<backup_name>
+   ```
+
+3. To delete the cStor backup completed jobs, use the following command.
+
+   ```
+   kubectl delete cstorbackupcompleted -n <backup_namespace> -l openebs.io/backup=<backup_name>
+   ```
+
+The deletion of Velero backup schedule doesn't destroy the backup created during the schedule. User need to delete a scheduled backup manually. Use the above steps to delete the scheduled backups.
+
+#### Troubleshooting guide for Backup and Restore
+
+1. Backup having status as ``FailedValidation``.
+
+```
```

Comment:
Add some space to left to include the content in list. 

Please do the same for all the list component